highfidelity/hifi
# Edit handles trigger fixes

Use a click handler on Controller.Standard.RTClick/LTClick inputs for consistency with trigger clicks in other controllers instead of specific trigger threshold values. Prevent both hands from triggering in entitySelectionTool at the same time.

Fixes: https://highfidelity.fogbugz.com/f/cases/14864/

Test Plan:
- Enter Interface in HMD mode on Rift
- Open Create app in Tablet and select any entity
- Point at different edit handles, slowly press trigger down to verify that it doesn't trigger the edit handle until the laser turns red, and verify the edit handles work as expected
- Point at the entity itself (outside of any edit handles) to perform XZ-plane translation, slowly press trigger down to verify that it doesn't trigger the translation until the laser turns red, and verify XZ-plane translation works as expected
- Point and trigger on any edit handle again, and while holding that handle attempt to point and trigger with the other hand anywhere on the entity including the already-triggered handle, and verify the other hand has no impact when triggering on the entity or handles
- Point and trigger on any edit handle again, and while holding that handle attempt to point and trigger on other entities to change selection as well as into the void to clear selection, and verify the other hand has no impact when triggering on other entities or into the void
- Verify when not triggering with either hand that the selected entity can be changed as expected by triggering on another entity with either hand's laser, as well as that the selection can be cleared as expected by triggering into the void with either hand's laser
- Continue to stress test around the above cases as well as edit handles themselves to verify all handles work as expected and there are no other issues
- Repeat above tests in HMD mode on Vive
